### ISSUE TITLE
The value NaN for parameter MetricData.member.11.Value is invalid

### ISSUE THREAD

**Issue by sharq1** [sj26/sidekiq-cloudwatchmetrics#16]
I got this error and it contains little detail.

`Aws::CloudWatch::Errors::InvalidParameterValue: The value NaN for parameter MetricData.member.11.Value is invalid.`

It says that it errors out on member 11, which is probably the `Utilization` metric.

My guess is that the `process["busy"]` and `process["concurrency"]` were zero's - then we have this situation:
```ruby
0 / 0.to_f
=> NaN
```

The question is why and when they would report zero's?

```
Aws::CloudWatch::Errors::InvalidParameterValue: The value NaN for parameter MetricData.member.11.Value is invalid.
-16 non-project frames
1
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-core-3.114.0/lib/seahorse/client/plugins/raise_response_errors.rb" line 17 in call
2
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-core-3.114.0/lib/aws-sdk-core/plugins/jsonvalue_converter.rb" line 22 in call
3
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-core-3.114.0/lib/aws-sdk-core/plugins/idempotency_token.rb" line 19 in call
4
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-core-3.114.0/lib/aws-sdk-core/plugins/param_converter.rb" line 26 in call
5
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-core-3.114.0/lib/seahorse/client/plugins/request_callback.rb" line 71 in call
6
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-core-3.114.0/lib/aws-sdk-core/plugins/response_paging.rb" line 12 in call
7
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-core-3.114.0/lib/seahorse/client/plugins/response_target.rb" line 24 in call
8
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-core-3.114.0/lib/seahorse/client/request.rb" line 72 in send_request
9
File "/home/app/vendor/cache/ruby/2.7.0/gems/aws-sdk-cloudwatch-1.51.0/lib/aws-sdk-cloudwatch/client.rb" line 2928 in put_metric_data
10
File "/home/app/vendor/cache/ruby/2.7.0/gems/sidekiq-cloudwatchmetrics-2.0.0/lib/sidekiq/cloudwatchmetrics.rb" line 181 in block in publish
11
File "/home/app/vendor/cache/ruby/2.7.0/gems/sidekiq-cloudwatchmetrics-2.0.0/lib/sidekiq/cloudwatchmetrics.rb" line 180 in each
12
File "/home/app/vendor/cache/ruby/2.7.0/gems/sidekiq-cloudwatchmetrics-2.0.0/lib/sidekiq/cloudwatchmetrics.rb" line 180 in each_slice
13
File "/home/app/vendor/cache/ruby/2.7.0/gems/sidekiq-cloudwatchmetrics-2.0.0/lib/sidekiq/cloudwatchmetrics.rb" line 180 in publish
14
File "/home/app/vendor/cache/ruby/2.7.0/gems/sidekiq-cloudwatchmetrics-2.0.0/lib/sidekiq/cloudwatchmetrics.rb" line 64 in run
15
File "/home/app/vendor/cache/ruby/2.7.0/gems/sidekiq-6.2.1/lib/sidekiq/util.rb" line 43 in watchdog
16
File "/home/app/vendor/cache/ruby/2.7.0/gems/sidekiq-6.2.1/lib/sidekiq/util.rb" line 52 in block in safe_thread
```

**Comment by stympy** [comment]
This is fixed by #14

**Comment by zerobearing2** [comment]
Having this issue, any reason #14 isn't merged and release yet?

**Comment by sj26** [comment]
Thanks, this should be fixed in [version 2.4.0](https://github.com/sj26/sidekiq-cloudwatchmetrics/releases/tag/v2.4.0).
